login.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+session_start();
+//const USERLIST = ['username' => 'robertkieronreeves'];
+
+If (isset($_POST['Submit']) {
+    $logins = array('robertkieronreeves')
+
+
+} && isset($_POST['rainbowsandunicorns'])) {
```

Comment:
this should be in the other branch, can you remove it from this branch please?

Reply:
I have since deleted from this branch and re-committed to the other branch.